_includes/manuals/1.10/4.4.8.3_prov_windows.md

```diff
@@ -0,0 +1,167 @@
+
+- WAIK PXELinux
+  - Name: `WAIK PXELinux`
+  - Kind: PXE Linux
+
+##### Optional templates
+- WAIK joinDomain.ps1
+  - Name: `WAIK joinDomain.ps1`
+  - Kind: User Data
+
+- WAIK local users
+  - Name: `WAIK local users`
+  - Kind: Snippet
+
+__Note:__ This snippet creates extra users in the unattended stage.
+This may be very useful for debugging early stages of your deployment; since you
+can find yourself locked out of the newly provisioned host.
+
+Microsoft does not really care about password security in unattend.xml files; so it does not really matter if you use
+`<PlainText>true</PlainText>` or not. If you prefer the encoded form, you need to append the string `Password` to your user password and encode it to Base64. The following ruby function is an example, it creates the _encoded_ from of `P@55w0rd`:
+
+```
+Base64.encode64(Encoding::Converter.new("UTF-8", "UTF-16LE",:undef => nil).convert("P@55w0rd"+"Password")).delete!("\n").chomp
+```
+
+- WAIK extraFinishCommands
+  - Name: `WAIK extraFinishCommands`
+  - Kind: Snippet
+
+__Note:__ The commands here are executed at the last stage just before finishing host building.
+Make sure they get executed in a synchronous way (eg. do not run in background like msiexec).
+Otherwise the following reboot might kill them.
+
+- WAIK OU from host group
+  - Name: `WAIK OU from host group`
+  - Kind: Snippet
+
+__Note__: This snippet may be used to generate the computer OU from the host's host group and domain.
+
+_Example_: Given a host `example` in domain `ad.corp.com` and in host group `servers/windows/databases`.
+The snippet generates the OU path:
+`OU=databases,OU=windows,OU=servers,DC=ad,DC=corp,DC=com`. Optionally, set the host parameter `computerOuSuffix` to add some arbitrary OU at the end.
+
+##### Add installation media
+For each of your Windows versions add a new installation medium pointing to the root of the folder containing _boot_ and _sources_
+Eg, `http://winmirror.domain.com/pub/win81x64`. Assign them to your operating system.
+
+##### Modify partition table
+The default partition table is a simple `diskpart.exe` script. It will wipe `Disk 0`
+
+##### Define templates
+Link all the created templates as well as the installation media and partition table to the OS:
+
+1. Head to your OS, then provisioning
+2. Select the template from each kind from the drop down list
+3. In partition tables, select `WAIK default`
+4. In installation media, check the appropriate installation media added above.
+
+![Link templates to OS](/static/images/screenshots/4.4.8.3_prov_win_templates.png "Linking Windows 8 OS in Foreman")
+
+##### Add Parameters
+To render the the templates correctly, some parameters need to be added. They need to be available as Host Parameter. Most of them make the most sense as parameter on the the OS. Also, almost none are
+required and have defaults. For the most up to date desciption see the template itself.
+
+##### Important parameters
+1. Required
```

Comment:
These numbered sections don't quite work, it starts a new numbered list down at "Optional" because the entries below it aren't all indented by four spaces.